.github/workflows/backport.yml

```diff
@@ -0,0 +1,29 @@
+name: Backport PR to branch
+on:
+  issue_comment:
+    types: [created]
+  schedule:
+    # once a day at 13:00 UTC to cleanup old runs
+    - cron: '0 13 * * *'
+
+permissions:
+  contents: write
+  issues: write
+  pull-requests: write
+  actions: write
+
+jobs:
+  backport:
+    if: ${{ contains(github.event.comment.body, '/backport to') || github.event_name == 'schedule' }}
+    uses: dotnet/arcade/.github/workflows/backport-base.yml@main
```

Comment:
Pin this to an SHA so it doesn't change unexpectedly.

Reply:
I wouldn't, if the Microsoft team made and update to it, we'll take it. The workflow base is designed to be inherited and is inherited by many projects. So I trust if there is a change to it, it's done to improve it. If it breaks us, we'll fix it. But there isn't much than can break here since we are not overriding much